ModelUploader/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "ModelUploader": {
       "commandName": "Project",
-      "commandLineArgs": "-p C:\\Users\\karlh\\Scratch\\Git\\opendigitaltwins-building\\Ontology"
+      "commandLineArgs": "-p C:\\Users\\akshayj\\Documents\\GitHub\\willow\\Willow\\  "
```

Comment:
Could we please change this commandLineArgs to be more generic? e.g. 
-p Ontology\

Reply:
Done